synapse/server.py

```diff
@@ -789,3 +789,7 @@ def get_outbound_redis_connection(self) -> Optional["RedisProtocol"]:
     def should_send_federation(self) -> bool:
         "Should this server be sending federation traffic directly?"
         return self.config.send_federation
+
+    def should_send_read_receipts(self) -> bool:
+        "Should this server be sending read receipts onwards?"
+        return self.config.send_read_receipts
```

Comment:
I don't think we need an accessor here (`should_send_federation` is overridden in a subclass, so is special). you can just do `hs.config.send_read_receipts` at the call site.